ddsp/training/train_util.py

```diff
@@ -134,7 +142,8 @@ def train(data_provider,
           steps_per_summary=300,
           steps_per_save=300,
           save_dir='~/tmp/ddsp',
-          restore_dir='~/tmp/ddsp'):
+          restore_dir='~/tmp/ddsp',
+          early_stop_loss_value=None):
   """Main training loop."""
```

Comment:
Yah, if you could add just a preliminary docstring here for the args, then I'll follow up on it in a later PR.

Reply:
Okay, I added args description.